File: app/lead_options.py
"""
Central list of degrees, subjects, and currencies for the lead form.
Edit this file to add, remove, or reorder options; the form and admin UI will pick up changes.
"""

# Degree levels (must match form dropdown and API validation)
# For now: Bachelor's and Master's only. PhD, Diploma, etc. are planned for later.
DEGREES = (
    "Bachelor's",
    "Master's",
)

# Subject areas. Include "Other" if you want a free-text fallback in the form.
SUBJECTS = (
    "Computer Science",
    "Business",
    "Engineering",
    "Medicine / Health",
    "Law",
    "Arts & Humanities",
    "Sciences",
    "Social Sciences",
    "Education",
    "Other",
)

# Currencies for tuition (budget) amount
CURRENCIES = (
    "USD",
    "EUR",
    "INR",
    "GBP",
    "AED",
)

File: frontend/src/constants/leadOptions.js
/**
 * Central list of degrees, subjects, and currencies for the lead form and admin.
 * Edit this file to add, remove, or reorder options.
 */

/** Degree levels. For now: Bachelor's and Master's only. PhD, Diploma, etc. are planned for later. */
export const DEGREES = [
  "Bachelor's",
  "Master's",
]

/** Subject areas. Include "Other" for a free-text fallback in the form. */
export const SUBJECTS = [
  'Computer Science',
  'Business',
  'Engineering',
  'Medicine / Health',
  'Law',
  'Arts & Humanities',
  'Sciences',
  'Social Sciences',
  'Education',
  'Other',
]

/** Currencies for tuition (budget) amount */
export const CURRENCIES = ['USD', 'EUR', 'INR', 'GBP']
